test/config.ini.in

```diff
@@ -9,6 +9,7 @@
 SRCDIR=@abs_top_srcdir@
 BUILDDIR=@abs_top_builddir@
 EXEEXT=@EXEEXT@
+RPCAUTH=@abs_top_srcdir@/share/rpcauth/rpcauth.py
```

Comment:
Could do a rebase, since this is already in master?

Reply:
Done